content/chainguard/libraries/python/build-configuration.md

```diff
@@ -99,6 +100,14 @@ for accessing your organization's Sonatype Nexus repository group.
    authentication is required, you must use the relevant details such as
    username and password in your build tool configuration.
 
+### Direct access
+
+Build configuration to retrieve artifacts directly from the Chainguard Libraries
+for Python repository at `https://libraries.cgr.dev/python/` with the simple
+index at `https://libraries.cgr.dev/python/simple` requires authentication with
```

Comment:
```suggestion
index at `https://libraries.cgr.dev/python/simple`. This requires authentication with
```

Again, i'm not sure what this doc is telling me to begin with, so take these comments with a grain of salt.